engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/rest/HttpFlinkClient.scala

```diff
@@ -93,7 +92,10 @@ class HttpFlinkClient(config: FlinkConfig)(implicit backend: SttpBackend[Future,
           .reverse
       }
       .map(WithDataFreshnessStatus.fresh)
-      .recoverWith(recoverWithMessage("retrieve Flink jobs"))
+      .recoverWith {
+        logger.trace(s"Checking fetching scenario state $jobName with policy $freshnessPolicy")
```

Comment:
Why you moved `"Checking fetching scenario state $jobName with policy $freshnessPolicy"` log from metod start to failure handling?

Reply:
I want to trace from the begging how was looked call this method in case of error.